iot_base/views/iot_system_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="iot_system_tree" model="ir.ui.view">
+        <field name="name">iot.system.tree</field>
+        <field name="model">iot.system</field>
+        <field name="arch" type="xml">
+            <tree create="0" delete="0">
```

Comment:
Why is it not possible to create a new system? But one can create it from the device.

Reply:
Systems should be created by other modules, because the actions must be configured for each system.
It is an error that they can be created from devices. Now is fixed :wink: